wrapanapi/systems/container/podman.py

```diff
@@ -0,0 +1,110 @@
+from abc import ABCMeta
+from datetime import datetime
+
+from podman import PodmanClient
+from proto.utils import cached_property
+
+from wrapanapi.entities.base import Entity
+from wrapanapi.systems.base import System
+
+
+class PodmanContainer(Entity, metaclass=ABCMeta):
+    def __init__(self, system, raw=None, **kwargs):
+        """
+        Constructor for an PodmanContainer tied to a specific host.
+
+        Args:
+            system: a Podman system object
+            raw: Raw container object if already obtained, or None
+        """
+        self._name = kwargs.get("name")
+        self._id = kwargs.get("id")
```

Comment:
i think it could also be nice to track the image as well

Reply:
Though its not useful in our current need, I could manage to add to track the image now. It was easier.